app/models/user.rb

```diff
@@ -1,6 +1,8 @@
 class User < ApplicationRecord
   VALID_EMAIL_REGEX = Settings.regex.email
 
+  attr_accessor :remember_token
```

Comment:
`attr_accessor` -> thằng này là gì ? có tác dụng gì? vì sao cần define `remember_token`?

Reply:
`attr_accessor:` dùng để tạo method `getter` và `setter `ạ

`remember_token` để lưu lại `token` được sinh, `remember_token` (để lưu trữ trong cookie) mà không cần lưu trữ trong cơ sở dữ liệu. Giúp tránh được việc có sự can thiệp vào quá trình đăng nhập của người sử dụng. Tăng độ an toàn cho ứng dụng và bảo vệ cho website.